### DISCUSSION TITLE
4.0-Thor

### DISCUSSION THREAD

**Discussion by Nimdy** [Nimdy/Dedicated_Valheim_Server_Script#310] [General]
Updated menu system for Valheim Ashlands release and Valheim+ repo change support

**Full Changelog**: https://github.com/Nimdy/Dedicated_Valheim_Server_Script/compare/3.5-lokis-dance...4.0-Thor

<hr /><em>This discussion was created from the release <a href='https://github.com/Nimdy/Dedicated_Valheim_Server_Script/releases/tag/4.0-Thor'>4.0-Thor</a>.</em>

**Comment by talissondc** [comment]
When I install V+, I can't connect to the server
